ReviewSite/review/__description/review__description.css

```diff
@@ -1,6 +1,6 @@
 .review__description {
-    margin: 10px;
-    width: 40%;
+    margin: 0.7rem;
+    width: 45vw;
```

Comment:
don't take values from viewport, t can be tricky, if it's container it should be depending on a content inside, if no set it via px or rem